cockroachcloud/databases-page.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Databases Page
+summary: The Databases page of the CockroachDB Cloud Console provides details about databases configured, the tables in each database, and the grants assigned to each user.
+toc: true
+docs_area: manage
+---
+
+The **Databases** page of the {{ site.data.products.db }} Console allows you to create new databases and provides details of the following:
+
+- The databases configured.
+- The tables in each database and the indexes on each table.
+- The grants assigned to each user.
+
+To view this page, click **Databases** in the left-hand navigation of the {{ site.data.products.db }} Console.
+
+## Databases
+
+The **Databases** page shows:
+
+- Whether [automatic statistics collection](../{{site.versions["stable"]}}/cost-based-optimizer.html#table-statistics) is enabled for the cluster.
+- A list of the databases on the cluster.
+- The **Add database** button, which allows you to create a new database.
+
+The following information is displayed for each database:
+
+| Column        | Description                           |
+|---------------|---------------------------------------|
+| Databases     | The name of the database.             |                           
+| Tables        | The number of tables in the database. |
```

Comment:
```suggestion
| Column        | Description                             |
|---------------|-----------------------------------------|
| Database Name | The name of the database.               |                           
| Tables        | The number of tables in the database.   |
| Action        | Actions to edit or delete the database. |
```

Reply:
Not sure if this column is necessary to include:
![image](https://user-images.githubusercontent.com/24593519/187915452-4fddbc9a-cd69-4a1f-85c2-26097266ba0f.png)